pkg/cedar/intake/models/easi_lifecycle_cost.go

```diff
@@ -0,0 +1,11 @@
+package models
```

Comment:
I wonder if we're overusing the `models` package. Seems to be used in auto-generated code, here, and in `pkg/models`

It may not practically be causing any problems because everything's named differently, but I wonder if it makes sense to scope the package more, like `package intakemodels`

Reply:
I was wondering about that. Go convention seems to be to default to shorter names and rely on importers to use a different name locally in case of collisions, though. From [Effective Go](https://go.dev/doc/effective_go#package-names):

>  By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. Err on the side of brevity, since everyone using your package will be typing that name. **And don't worry about collisions a priori. The package name is only the default name for imports; it need not be unique across all source code, and in the rare case of a collision the importing package can choose a different name to use locally.**

I'm open to changing it, though.

Reply:
Nah, you're right. I forgot that the importer would have the full path, so it's not like the files I listed are actually part of the same package